contrib/buildahimage/centos7/Dockerfile

```diff
@@ -8,19 +8,32 @@ FROM centos:7
 
 # Remove directories used by yum that are just taking
 # up space.
-RUN useradd build; yum -y update; rpm --restore shadow-utils 2>/dev/null; yum -y install cpp buildah fuse-overlayfs xz; rm -rf /var/cache /var/log/dnf* /var/log/yum.*;
+RUN useradd build && \
+    yum -y update && \
+    yum -y reinstall shadow-utils && \
+    yum -y install buildah fuse-overlayfs xz && \
```

Comment:
Minor nit: Probably about time to switch these to `dnf`.

Reply:
Colin Walters and a few others were pretty much against that.  I took Colins word over mine.

Reply:
Ahh, hmmm.  I'd argue the opposite, everything of import uses `dnf` now, and it makes things more portable to use a common tool.  Anyway, this is merged, so :shrug: